### PR TITLE
A4A: Eliminate some runts in the Referrals FAQs page.

### PR DESCRIPTION
**Pull request by jkguidaven**
This PR eliminates the "runts" in the FAQs section titles.

| Before | After |
|--------|--------|
| Screenshot 2024-06-20 at 8 44 26 PM | Screenshot 2024-06-20 at 9 02 01 PM | 

Closes https://github.com/Automattic/jetpack-genesis/issues/402

## Proposed Changes

* Insert `&nbsp;` to ensure titles do not produce runts.

## Testing Instructions

* Use the A4A live link and go to `/referrals/faq` page
* confirm that the title do not have runts.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/referrals/primary/commission-overview/index.tsx [hunk 1]
```diff
@@ -106,7 +106,15 @@ export default function CommissionOverview( {
 								<div className="commission-overview__heading">
 									<img src={ pressableIcon } alt="Pressable" />
 									<WordPressLogo className="a4a-overview-hosting__wp-logo" size={ 24 } />
-									{ translate( 'Hosting revenue share (WordPress.com and Pressable)' ) }
+									{ translate(
+										'Hosting revenue share{{br/}}{{nbsp/}}(WordPress.com and Pressable)',
+										{
+											components: {
+												br: <br />,
```
Comment (yashwin) [inline client/a8c-for-agencies/sections/referrals/primary/commission-overview/index.tsx]: I don't think we need a break here. I think the idea is not to keep the last word hanging.

I think it should be 

- and{{nbsp/}}Pressable)
- Woo-owned{{nbsp/}}extensions

@madebynoam

Comment (jkguidaven) [reply client/a8c-for-agencies/sections/referrals/primary/commission-overview/index.tsx]: Thanks. Made some adjustment.

**Review by yashwin — APPROVED** [review]
LGTM 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/15015273

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @jkguidaven for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/15015273).

**Pull request merged**